src/integrationTest/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterTest.java

```diff
@@ -49,16 +49,16 @@
 
     @Test
     public void should_return_200_when_single_letter_is_sent() throws Throwable {
```

Comment:
We are using Around Advice which means it gets invoked before and after the method call.It will good if that is evident in the test case.